app/scripts/directives/podMetrics.js

```diff
@@ -98,6 +100,20 @@ angular.module('openshiftConsole')
             ]
           });
         }
+        generateVolumeDataSets();
+        if (_.includes(scope.includedMetrics, "volumes")) {
```

Comment:
In my opinion, pod share the persistent volumes with its replications. I found  it take "Downsampling" to get all deployment metrics to makes the user experience better. Pod mabye have many pvcs.